content/docs/start/data-pipelines.md

```diff
@@ -1,13 +1,24 @@
-# Data Pipelines
+---
+title: 'Get Started: Data Pipelines'
+description: 'Get started with pipelines in DVC. Learn how to capture, organize,
+version, and reproduce your data science and machine learning workflows.'
+---
+
+# Get Started: Data Pipelines
 
 Versioning large data files and directories for data science is great, but not
 enough. How is data filtered, transformed, or used to train ML models? DVC
 introduces a mechanism to capture _data pipelines_ — series of data processes
 that produce a final result.
 
-DVC pipelines and their data can also be easily versioned (using Git). This
-allows you to better organize your project, and reproduce your workflow and
-results later exactly as they were built originally!
+A DVC pipeline uses stages to describe how models and other data artifacts are
+built, and provides an efficient way to reproduce them. This allows you to
+capture simple data workflows, organize data science projects, or build detailed
+machine learning pipelines (to name a few use cases).
+
+You can easily version DVC pipelines and their data (using Git). This allows you
+to better organize your project and reproduce your workflow and results later —
+exactly as they were built originally!
```

Comment:
Simple workaround to https://github.com/iterative/dvc.org/pull/1857#pullrequestreview-509809200:

```suggestion
DVC pipelines and their data can also be easily versioned (using Git). This
allows you to better organize your project, and reproduce your workflow and
results later exactly as they were built originally! This allows you to
capture simple data workflows, organize data science projects, or build detailed
machine learning pipelines (to name a few use cases).
```
> Needs formatting